### DISCUSSION TITLE
Global settings for a plugin

### DISCUSSION THREAD

**Discussion by jphalip** [elgatosf/elgato#48] [Help] [answered]
Hi,

Is there a way to have a user-editable global setting that would be shared by all actions of a plugin. In other terms, the user would only need to set a value once in the Stream Deck app for the plugin, and then all instances of the plugin actions would be able to retrieve that single value when their keyUp event is triggered.

Hope that makes sense. Thanks!

Julien

**Comment by tiptronic** [comment, marked as answer]
Hi @jphalip 

please check this:

https://docs.elgato.com/sdk/plugins/events-sent#setglobalsettings

also see my answer [here](https://github.com/orgs/elgatosf/discussions/53#discussioncomment-6677420) how to use it
